### PR TITLE
feat: Single view of a restaurant

### PR DESCRIPTION
**Pull request by ashikka**

### CODE REVIEW

**Comment by gornovah** [comment]
I think this file is here for error, isn't it? assets/CSE1007-Java Programming-Quiz 2-A2 slot.pdf @ashikka

File: views/restaurantView.tsx [hunk 1]
```diff
@@ -0,0 +1,293 @@
+import React, { useEffect } from "react";
+import { useDispatch, useSelector } from "react-redux";
+import { StyleSheet, View, ImageBackground, Text, Image } from "react-native";
+import Icon from "react-native-vector-icons/FontAwesome5";
+import { RootState } from "../redux/store";
+import { getRestaurantDetails } from "../api/requests";
```
Comment (gornovah) [inline views/restaurantView.tsx:6]: This import is not used, right?

File: views/restaurantView.tsx [hunk 2]
```diff
@@ -0,0 +1,293 @@
...
+  }
+
+  useEffect(() => {
+    getRestaurant();
+  }, [dispatch]);
+
+  if (restaurant.restaurant.attributes) {
+    console.log(restaurant.restaurant.attributes.store);
+  }
+
```
Comment (gornovah) [inline views/restaurantView.tsx:102]: You could move the different views to functions, names that make sense of what they do, what do you think?
  
function getViewNameStore() {
    return <View style={{flexDirection: "row", alignContent: "center"}}>
      <Text style={styles.rating}>
        {restaurant.restaurant.attributes.store.name}
      </Text>
      <Icon style={styles.rating} name="star" color="gold" solid/>
      <Text style={styles.rating}>
        {restaurant.restaurant.attributes.pack.rating}
      </Text>
    </View>;
  }

File: views/restaurantView.tsx [hunk 3]
```diff
@@ -0,0 +1,293 @@
...
+            <View>
+              <Text>
+                ${" "}
+                {
+                  restaurant.restaurant.attributes.pack
+                    .unitary_price_after_discount.value
+                }
+              </Text>
+            </View>
+          </View>
```
Comment (gornovah) [inline views/restaurantView.tsx]: {getViewNameStore()} call in the first comment

**Comment by gornovah** [comment]
Can you update the readme of the project? it is very good when we develop to have a good readme. 
[README.md](https://github.com/code-sherpas/too-good-to-go-client/files/7660464/README.md)

File: views/restaurantView.tsx (hunk 1 above)
Comment (ashikka) [reply views/restaurantView.tsx:6]: No, they are used

File: views/restaurantView.tsx (hunk 2 above)
Comment (ashikka) [reply views/restaurantView.tsx:102]: Done!

**Review by gornovah — APPROVED** [review]

**Pull request merged**